pkgs/development/tools/rust/cargo-show-asm/default.nix

```diff
@@ -1,13 +1,16 @@
 { lib
-, stdenv
-, fetchFromGitHub
 , rustPlatform
-, pkg-config
+, fetchFromGitHub
+, curl
 , installShellFiles
+, pkg-config
 , openssl
+, stdenv
+, darwin
```

Comment:
I'm against reordering the existing one before a tree-wide enforced formatting spec.

Reply:
even though not strictly enforced, there is a sorting convention for nixpkgs

> Arguments should be listed in the order they are used, with the exception of lib, which always goes first.

https://nixos.org/manual/nixpkgs/unstable/#sec-syntax